internal/common/common.go

```diff
@@ -241,3 +244,45 @@ func GetHostNTPSources(db *gorm.DB, host *models.Host) (string, error) {
 	}
 	return infraEnv.AdditionalNtpSources, nil
 }
+
+func ParsePemCerts(pemCerts []byte) ([]x509.Certificate, bool) {
+	var certs []x509.Certificate
+	for len(pemCerts) > 0 {
```

Comment:
`for len(pemCerts) > 0` ? -- is it decreasing or something?
maybe you meant `if` ?

Reply:
I updated the function to be more readable